docs/configuration.md

```diff
@@ -2309,6 +2309,14 @@ Apart from these, the following properties are also available, and may be useful
   </td>
   <td>0.6.0</td>
 </tr>
+<tr>
+  <td><code>spark.speculation.min.threshold</code></td>
+  <td>100ms</td>
+  <td>
+    Minimum amount of time a task runs before being considered for speculation.
```

Comment:
sorry I wasn't clear, I meant for you to keep the second sentence as well to explain its use.

Maybe something like: "
Minimum amount of time a task runs before being considered for speculation.
This can be used to avoid launching speculative copies of tasks that are very short."

Reply:
ok